docs/news/1292.bugfix

```diff
@@ -0,0 +1,2 @@
+If autostart is disabled, a CloudException is no longer raised when there is no long-polling thread.
```

Comment:
It would be worth giving the context that this is for async notifications retrieve resource values